src/Makefile

```diff
@@ -308,6 +317,25 @@ endif
 
 $(LIBCYPHER_PARSER_DIR)/cypher-parser.h : $(LIBCYPHER_PARSER)
 
+# Set runConfigureICU os choice
+ifeq ($(OS),macos)
+ICU_CONFIG_OS = MacOSX
+else ifeq ($(OS),linux)
+ICU_CONFIG_OS = Linux
+endif
+# configuration for UNIX system
+$(ICU4C_LIBS):
+ifeq (,$(wildcard $(ICU4C_LIBS)))
+	@mkdir -p $(ICU4C_MAKE_DIR) ;\
+	cd $(ICU4C_MAKE_DIR) ;\
+	chmod +x $(ICU4C_SRC_DIR)/runConfigureICU $(ICU4C_SRC_DIR)configure $(ICU4C_SRC_DIR)install-sh ;\
+	CPPFLAGS="-fPIC -DU_NO_DEFAULT_INCLUDE_UTF_HEADERS=1 -DUCONFIG_ONLY_COLLATION=1" \
+	$(ICU4C_SRC_DIR)/runConfigureICU $(ICU_CONFIG_OS) --enable-static --disable-shared \
+	prefix=$(ICU4C) ;\
+	$(MAKE);\
+	$(MAKE) install
```

Comment:
Why do we to `make install` isn't `make` enough ?

Reply:
I am following the instruction [here](https://unicode-org.github.io/icu/userguide/icu4c/build.html#how-to-build-and-install-on-unix). I think `make` compiles the files while `make install` copies the compiled files to the predefined directory.